### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by Wang-Siyuan** [bigmessyteam/orie4741#11]
The objective of this project is to predict whether a hotel room may be booked by a customer as well as the GBU of the booking.

Highlights:
- I really like the fact that you took the time to examine each feature and do some initial screening before trying out a subset of features. The histogram, bar graph and other visualization of the data are also concise and useful for determining whether or not it makes sense to include them in the model.
- I appreciate the fact that you tried out different feature space and comparing them by plotting their accuracy. I especially like the third model you tried, which involved more complex feature space. The process of trying different model might be frustrating, but it's great to see you are actively exploring your options.
- It was great that you were using what we learned in class to avoid overfitting by using regularizer and to use cross-validation to find the best parameter for regularization.
- I also liked the fact that you went the extra mile to explore how to solve the issue you faced in Julia by switching to Python.  

Rooms for improvement/clarification:
- The proposal of your project states that you wanted to build a recommendation system to suggest hotel booking when a given customer enters a search.  However, you didn't mention specifically why you think GBU will be beneficial for building the recommendation system. I assume you might have done some research on why this can help you build the recommendation system, but it would be helpful to explain your reasoning to the reader.

- There are many assumptions about the input data that needs to be clarified.  For example, are you restricting the location of the hotel? Of course, the Kaggle dataset might only have hotel booking information for hotels that are close by, because it's based on the customer's booking history.  But if you are recommending hotels to users, you have to take into account factors such as the location.

- Most of your existing work are focusing on regression for predicting GBU, but at the last section, you mentioned "Since the Result is a boolean value".  I assume you also want to work on the classification problem where you want to predict whether the booking will occur. But you should make it clear about the two approaches you are taking and how they could both contribute to your recommendation system.

- Regarding the training set/validation set/test set, it would be great if you could discuss how you divided the data set into these groups.
